infra/main.bicep

```diff
@@ -66,12 +66,18 @@ param embeddingDeploymentName string // Set in main.parameters.json
 param embeddingDeploymentCapacity int = 30
 param embeddingModelName string = 'text-embedding-ada-002'
 
+param tenantId string = tenant().tenantId
+param authTenantId string = ''
+
 // Used for the optional login and document level access control system
 param useAuthentication bool = false
+param enforceAccessControl bool = false
 param serverAppId string = ''
 @secure()
 param serverAppSecret string = ''
 param clientAppId string = ''
+@secure()
+param clientAppSecret string = ''
```

Comment:
This needs to be stored in Key Vault

Reply:
will update when we have key vault support